src/components/Header.tsx

```diff
@@ -0,0 +1,51 @@
+import React, { useContext, useState } from 'react';
+import cn from 'classnames';
+import { Todo } from '../types/Todo';
+import { ErrorContext, ErrorsMessageContext } from './ErrorsContext';
+
+type Props = {
+  todos: Todo[];
+};
+
+export const Header: React.FC<Props> = ({ todos }) => {
+  const { setErrorsMesage } = useContext(ErrorsMessageContext);
+  const [value, setValue] = useState('');
+  const { setIsError } = useContext(ErrorContext);
+  const addTodo = (e: React.FormEvent) => {
+    e.preventDefault();
+    if (value.length <= 0) {
+      setIsError(true);
+      setErrorsMesage('empty');
+    }
+  };
+
+  return (
+    <header className="todoapp__header">
+      {todos?.length && (
```

Comment:
```suggestion
      {!!todos?.length && (
```
to fix this
<img width="600" alt="image" src="https://github.com/mate-academy/react_todo-app-loading-todos/assets/70220667/8c3a5136-7e3b-456e-b32b-4b66abfb98bc">